plugins/chat/spec/system/mention_warnings_spec.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+describe "when a mention won't create a notification", type: :system, js: true do
+  fab!(:user) { Fabricate(:user) }
+  fab!(:user_2) { Fabricate(:user) }
+  fab!(:channel_1) { Fabricate(:chat_channel) }
+
+  let(:chat_page) { PageObjects::Pages::Chat.new }
+  let(:chat_channel_page) { PageObjects::Pages::ChatChannel.new }
+
+  before do
+    chat_system_bootstrap(user, [channel_1])
+    sign_in(user)
+  end
+
+  fab!(:admin_mentionable_group) { Fabricate(:group, mentionable_level: Group::ALIAS_LEVELS[:only_admins]) }
+  fab!(:publicly_mentionable_group) { Fabricate(:group, mentionable_level: Group::ALIAS_LEVELS[:everyone]) }
```

Comment:
With what I just said after in mind, we can now create these groups only when needed in the right context and just call them "group" as it will be clear from the context what kind of group it is.